inc/saiswitch.h

```diff
@@ -1052,8 +1057,9 @@ typedef enum _sai_switch_attr_t
      * @brief The hash object for IPv6 packets going through LAG
      *
      * @type sai_object_id_t
-     * @flags READ_ONLY
+     * @flags CREATE_AND_SET
      * @objects SAI_OBJECT_TYPE_HASH
+     * @allownull true
```

Comment:
Yea so that default hash is the internal prezent value do we should nie allow null but set that internal value to bring back default